openapi/multi-yaml/index.yaml

```diff
@@ -79,6 +79,223 @@ tags:
+        "payment_method_group_id": "pmg_xyz123",
+        "statement_descriptor": "Big Business"
+      }
+      ```
+
+      ##### Example Response: Checkout Created
+      ```json
+      {
+        "id": "cho_123xyz",
+        "type": "checkout",
+        "page_info": null,
+        "data": {
+          "id": "cho_123xyz",
+          "payment_intent_id": null,
+          "account_id": "acc_subacc123",
+          "platform_account_id": "acc_plat123",
+          "payment_amount": 10000,
+          "payment_currency": "USD",
+          "payment_description": "",
+          "payment_methods": [],
+          "payment_method_group_id": "pmg_xyz123",
+          "status": "created",
+          "mode": "test",
+          "payment_settings": {
+            "credit_card_payments": true,
+            "ach_payments": true,
+            "bnpl_payments": false,
+            "insurance_payments": false
+          },
+          "successful_payment_id": null,
+          "statement_descriptor": null,
+          "application_fees": null,
+          "created_at": "2024-10-29T21:22:02.857Z",
+          "updated_at": "2024-10-29T21:22:02.857Z",
+          "completions": []
+        }
+      }
+      ```
+
+      ##### Example Request: Complete Checkout
+      ```json
+      {
+        "payment_token": "pm_123walletpm"
+      }
+      ```
+
+      ##### Example Response: Checkout Completed
+      ```json
+      {
+        "id": "chc_4KIFTRC1hmdQx1X6jssYLk",
+        "type": "checkout_completion",
+        "page_info": null,
+        "data": {
+          "payment_mode": "ecom",
+          "payment_token": "pm_autoclonedxyz789",
+          "payment_status": "succeeded",
+          "payment_error_code": null,
+          "payment_error_description": null,
+          "checkout_id": "cho_123xyz",
+          "payment_id": "py_123xyz",
+          "payment_method_id": "pm_autoclonedxyz789",
+          "status": "succeeded",
+          "created_at": "2024-10-29T21:25:36.692Z",
+          "updated_at": "2024-10-29T21:25:36.692Z",
+          "payment_response": {
+            "id": "py_123xyz",
+            "type": "payment",
+            "data": {
+              "amount": 10000,
+              "currency": "usd",
+              "status": "succeeded",
+              "payment_method": {
+                "card": {
+                  "id": "pm_autoclonedxyz789",
+                  "acct_last_four": "3222",
+                  "brand": "mastercard",
+                  "name": "Tifa Lockhart",
+                  "token": "pm_autoclonedxyz789"
+                }
+              }
+            }
+          }
+        }
+      }
+      ```
+
+      Note that when completing the checkout with a wallet payment method token, the system automatically creates a cloned payment method for the sub-account as shown in the response's payment_method.token field. 
+      This same token will be retrieved for subsequent payment requests for the same Sub-Account passing the same wallet token
```

Comment:
Would you mind adding the checkout web component example as an alternative way of completing the checkout? It could be similar to the PaymentForm web component inclusion above.